permissions/plugin-portshift-scanner.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "portshift-scanner"
+github: "jenkinsci/portshift-ci-plugin"
```

Comment:
Sorry, missed one thing, the GitHub repo needs to be updated

Reply:
fixed as suggested